src/Propulsion.Feed/PeriodicSource.fs

```diff
@@ -38,7 +38,7 @@ module private TimelineEvent =
         fun (i, x: FsCodec.IEventData<_>, context: obj) ->
             if i > DateTimeOffsetPosition.factor then invalidArg (nameof i) $"Index may not exceed %d{DateTimeOffsetPosition.factor}"
             FsCodec.Core.TimelineEvent.Create(
-                baseIndex + i, x.EventType, x.Data, x.Meta, x.EventId, x.CorrelationId, x.CausationId, x.Timestamp, isUnfold = true, context = context)
```

Comment:
whoopsie?

Reply:
no, it's intentional - this first/only use of the flag was to convey that these were ephemeral events, not backed by a store
but nothing leans on that
the change in this PR as a whole is that IsUnfold events now get dropped and replaced whenever a new set comes through (similar to how cosmos changefeed will deliver only the latest state)
But that would be bad for periodic source, which generates a set of sequntial events to represent the full state of the input at a point in time (and then re-runs the search with a new prefix when the period resets)